superset/views.py

```diff
@@ -2026,6 +2026,42 @@ def fave_dashboards(self, user_id):
 
     @api
     @has_access_api
+    @expose("/fave_queries/", methods=['GET'])
+    def fave_queries(self):
+        qry = (
+            db.session.query(
+                models.Query,
+            )
+            .join(
+                models.FavStar,
+                sqla.and_(
+                    models.FavStar.user_id == int(g.user.id),
+                    models.FavStar.class_name == 'query',
+                    models.Query.client_id == models.FavStar.obj_id,
+                )
+            )
+            .order_by(
+                models.FavStar.dttm.desc()
+            )
+        )
+        queries = [q.to_dict() for q in qry.all()]
```

Comment:
it would be nice just to pass queries and use them in the frontend, other approach could be to add an additional param to the to_dict function where you would pass fields you care about